src/tree_view.ts

```diff
@@ -0,0 +1,414 @@
+
+import { spawn } from 'child_process';
```

Comment:
We're aiming to be able to run as a VSCode Web Extesion (#1) so using node-js specific APIs should be avoided.

From https://code.visualstudio.com/api/extension-guides/web-extensions#web-extension-main-file

> Node.js globals and libraries such as process, os, setImmediate, path, util, url are not available at runtime.